src/components/ebay-progress-spinner/index.marko

```diff
@@ -0,0 +1,8 @@
+import processHtmlAttributes from "../../common/html-attributes"
+
+static var ignoredAttributes = [
+    "a11yText",
+];
+
+$ var label = input.a11yText ? input.a11yText : "Busy";
+<span ...processHtmlAttributes(input, ignoredAttributes) class=[ "spinner", input.size === "large" && "spinner--large"] aria-label=label role="img"></span>
```

Comment:
Make sure you add `input.class` to the `class` array so users can add their own classes as needed.